<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="figures_list">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>0</y>
      <width>200</width>
      <height>691</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>790</width>
      <height>690</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>740</width>
      <height>690</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>396</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>60</height>
     </size>
    </property>
    <property name="title">
     <string>Цвет границы</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="border_color_preview" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_border_color_change">
        <property name="text">
         <string>Поменять</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>396</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>60</height>
     </size>
    </property>
    <property name="title">
     <string>Цвет заполнения</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="fill_color_preview" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fill_color_change">
        <property name="text">
         <string>Поменять</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>396</width>
      <height>70</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>391</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="is_delay">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Заполнение с задержкой</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Время задержки(мс)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="delay_time">
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>296</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_x"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_y"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_add_dot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>379</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить точку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close_figure">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>379</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Замкнуть фигуру</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_fill_dot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>379</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить затравку</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Затравка (x; y): Не задана</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>381</width>
      <height>165</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,2,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Координаты центра (x:y)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="center_x">
         <property name="text">
          <string>400</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="center_y">
         <property name="text">
          <string>350</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Окружность</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Эллипс</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Радиус (R)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="radius"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Ширина (Rx)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="radius_x"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Высота (Ry) </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="radius_y"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="btn_circle_draw">
         <property name="text">
          <string>Построить окружность</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ellipse_draw">
         <property name="text">
          <string>Построить эллипс</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_remove_fill">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>608</y>
      <width>396</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Убрать заполнение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fill">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>566</y>
      <width>396</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить заполнение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>396</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить все</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
